src/components/tab-nav/tab-nav.tsx

```diff
@@ -223,6 +223,26 @@ export class TabNav {
     event.preventDefault();
   }
 
+  @Listen("calciteInternalTabsFocusFirst")
+  focusFirstTabHandler(event: CustomEvent): void {
```

Comment:
Can you extract the common logic between the event handlers into a separate method?